coderd/rbac/scopes.go

```diff
@@ -52,12 +62,12 @@ func WorkspaceAgentScope(params WorkspaceAgentScopeParams) Scope {
 		// Limit the agent to only be able to access the singular workspace and
 		// the template/version it was created from. Add additional resources here
 		// as needed, but do not add more workspace or template resource ids.
-		AllowIDList: []AllowListElement{
+		AllowIDList: append([]AllowListElement{
 			{Type: ResourceWorkspace.Type, ID: params.WorkspaceID.String()},
 			{Type: ResourceTemplate.Type, ID: params.TemplateID.String()},
 			{Type: ResourceTemplate.Type, ID: params.VersionID.String()},
 			{Type: ResourceUser.Type, ID: params.OwnerID.String()},
-		},
+		}, extraAllowList...),
```

Comment:
This is nice and narrow :+1: 


Just for future info, you can also do this: `{Type: ResourceTask.Type, ID: policy.WildcardSymbol},`

To give it access to all tasks. I assume each workspace just has 1 task though. And being narrow is better :+1:

Reply:
Yep, tasks and workspaces are 1:1 currently. Good to know about the wildcard though, thanks. 👍🏻